docs/intro/readnext.md

```diff
@@ -0,0 +1,31 @@
+# What to read next
+
+So you’ve read all the introductory material and have decided you’d like to keep using Krustlet to run your WebAssembly
+applications. We’ve only just scratched the surface.
+
+So what’s next?
+
+Well, we’ve always been big fans of learning by doing. At this point you should know enough to start a project of your
+own and start fooling around. As you need to learn new tricks, come back to the documentation.
+
+We’ve put a lot of effort into making Krustlet's documentation useful, easy to read and as complete as possible. The
+rest of this document explains more about how the documentation works so that you can get the most out of it.
+
+(Yes, this is documentation about documentation. Rest assured we have no plans to write a document about how to read the
+document about documentation.)
+
+## How the documentation is organized
+
+Krustlet's main documentation is broken up into “chunks” designed to fill different needs:
+
+The [introductory material][intro] is designed for people new to Krustlet – or to WebAssembly on Kubernetes in
+general. It doesn’t cover anything in depth, but instead gives a high-level overview of how running apps on Kubernetes
+“feels”.
+
+The [topic guides][topics], on the other hand, dive deep into individual parts of Krustlet. There are complete guides to
+the internals of Krustlet's architecture. This is probably where you’ll want to spend most of your time; if you work
+your way through these guides you should come out knowing pretty much everything there is to know about Krustlet.
+
+
+[introductory material]: README.md
```

Comment:
Are these supposed to be links?

Reply:
Yes. This is valid markdown syntax! It's a [link label](https://daringfireball.net/projects/markdown/syntax#link), and it's used above on line 21 as a reference-style link. This was accidentally renamed though. Fixing that now